ibmdbpy/statistics.py

```diff
@@ -637,8 +637,8 @@ def cov(idadf, other = None):
         raise TypeError("cov() missing 1 required positional argument: 'other'")
 
     columns = idadf._get_numerical_columns()
-    if not columns:
-        print(idadf.name + " has no numeric columns")
+    if not columns or len(columns) < 2:
```

Comment:
Please remove the 3 tests marked `xfail` in https://github.com/ibmdbanalytics/ibmdbpy/blob/master/ibmdbpy/tests/test_statistics.py#L190-L193. They are now not failing by accident, but by design.

Reply:
They succeeded:
``
test_statistics.py::Test_DescriptiveStatistics::test_idadf_statistics_one_column[cov] XPASS [ 96%]
test_statistics.py::Test_DescriptiveStatistics::test_idadf_statistics_one_column[corr] XPASS [ 96%]
test_statistics.py::Test_DescriptiveStatistics::test_idadf_statistics_one_column[quantile] PASSED [ 96%]
test_statistics.py::Test_DescriptiveStatistics::test_idadf_statistics_one_column[mad] XPASS [ 96%]
``
So why should we remove them?

Reply:
Oh, right, the only indication of an error in cov/corr/mad is the message that's being printed. That makes the tests pretty much useless, but means you can at least remove the `xfail` marker from them. That will change their result from `XPASS` (expected to fail, but passed) to just `PASS`

Reply:
Changed with commit https://github.com/ibmdbanalytics/ibmdbpy/pull/56/commits/6d8043c019bcf089e107474b3f698a0c3ba976f3